helpers/calendar/analytics_helper.py

```diff
@@ -38,6 +38,23 @@ def convert_dates(self, start_date, end_date):
         end_date = (datetime.strptime(end_date, "%b %d %Y") + relativedelta(days=1)).isoformat() + 'Z'  # noqa: E501
         return(start_date, end_date)
 
+    def validate_current_date(self, start_date, end_date):
```

Comment:
From what I'm seeing you're not necessarily validating the current date but rather the date range input by the user. Kindly clarify.

Reply:
Hi @bev what this function does is that it defines the current date and checks if non-of the dates provided exceeds the current date. Kindly give an idea of how this function can be named better. I could use `validate_dates` but I thought that wouldn't bring out what the function does well.

Reply:
@nakatuddesuzan have you thought of using/or extending the `validate_date_time_range` function in `utilities/validations.py`?

Reply:
Hello @joshuaocero , no I hadn't  thought of it, Let me try it out, thanks

Reply:
Hello @joshuaocero , I have tried it out and it seems not to be working. From the research, I have made, It shows that the developers before decided to have methods to do with analytics in their folder. So shifting that method to `validations.py` would lead to a lot of failures. So it's better to leave that method in the `analytics_helper.py`